airbyte-integrations/bases/bases-destination-jdbc/src/main/java/io/airbyte/integrations/destination/staging/StagingConsumerFactory.java

```diff
@@ -112,7 +141,7 @@ public SerializedAirbyteMessageConsumer createAsync(final Consumer<AirbyteMessag
         () -> GeneralStagingFunctions.onCloseFunction(database, stagingOperations, writeConfigs, purgeStagingData, typerDeduper).accept(false),
         flusher,
         catalog,
-        new BufferManager(),
+        BufferMemoryLimit.map(memoryLimit -> new BufferManager(memoryLimit)).orElse(new BufferManager()),
```

Comment:
can we put this in a variable or method for clarity

Something like 
`bufferMemoryLimit.getOrElse(DEFAULT_VALUE)`

Reply:
Done